tests/integration_tests/test_storage_migration.py

```diff
@@ -59,3 +95,19 @@ def test_that_storage_matches(tmp_path, source_root, snapshot, monkeypatch):
             snapshot.assert_match(
                 str(experiment.response_configuration) + "\n", "responses"
             )
+
+            summary_data = ensemble.load_responses(
+                "summary",
+                tuple(ensemble.get_realization_list_with_responses("summary")),
+            )
+            snapshot.assert_match(
+                summary_data.to_dataframe().astype(np.float32).to_csv(),
```

Comment:
Is the ordering the summary data meant to be locked in? If not, could you add a sorting to the dataframe to make the test insensitive to row order? `.transform(np.sort)` or something liek that. With the way summary data is used now, AFAIK the order does not matter, only the contents